### PR TITLE
Add test for new stream functionality

### PR DESCRIPTION
**Pull request by hollandben**
Follows on from #49 

I decided to test it via `pino` itself so that we can be sure the integration is working properly.

### CODE REVIEW

File: tests/tests.js [hunk 1]
```diff
@@ -54,3 +57,70 @@ test('pino-stackdriver adds severity to log entry', ({ expect }) => {
     cp.stdin.push(null)
   })
 })
+
+test('pino-stackdriver adds severity to log entry via createStream',
+  ({ expect }) => {
+    return new Promise(resolve => {
+      const pid = process.pid
+      const hostname = os.hostname()
+      const originaDateNow = Date.now
```
Comment (ianwalter) [inline tests/tests.js]:
```suggestion
      const originalDateNow = Date.now
```

File: tests/tests.js [hunk 2]
```diff
@@ -54,3 +57,70 @@ test('pino-stackdriver adds severity to log entry', ({ expect }) => {
...
+            .split('\n')
+            .filter(line => line)
+            .map(JSON.parse)
+          lines = lines.concat(line)
+          next()
+        }
+      })
+
+      stdout.on('close', () => {
+        Date.now = originaDateNow
```
Comment (ianwalter) [inline tests/tests.js]:
```suggestion
        Date.now = originalDateNow
```

File: index.js [hunk 3]
```diff
@@ -40,8 +40,12 @@ function pinoStackdriver (line) {
 
 const transform = split(pinoStackdriver)
 
-function createStream () {
-  return pumpify(transform, process.stdout)
+function createStream (dest) {
```
Comment (ianwalter) [inline index.js]:
```suggestion
function createStream (dest = process.stdout) {
```
(instead of logic below)

File: tests/tests.js [hunk 4]
```diff
@@ -1,7 +1,10 @@
 const { join } = require('path')
-const { Readable } = require('stream')
+const os = require('os')
+const { Readable, Writable } = require('stream')
 const { test } = require('@ianwalter/bff')
 const execa = require('execa')
+const pino = require('pino')
+const createStream = require('../index').createStream
```
Comment (ianwalter) [inline tests/tests.js]:
```suggestion
const { createStream } = require('../index')
```

**Comment by hollandben** [comment]
I've also updated the README with some more information on the API for the module (using the structure from Pino's readme)

**Pull request merged**